sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -60,4 +61,31 @@ export class InstitutionController extends BaseController {
   async sync(@UserToken() token: IUserToken) {
     await this.institutionService.syncInstitution(token);
   }
+
+  @Get("/users")
+  async allUsers(
+    @UserToken() user: IUserToken,
+  ): Promise<InstitutionUserRespDto[]> {
+    const institution = await this.institutionService.getInstituteByUserName(
+      user.userName,
+    );
+    return (await this.institutionService.allUsers(institution.id)).map(
+      (item) => {
+        const r: InstitutionUserRespDto = {
```

Comment:
It will be better to rename the constant "r" to something more meaningful.

Reply:
👍